lib/order.rb

```diff
@@ -0,0 +1,43 @@
+class Order
+  
+  attr_reader :total
+  attr_accessor :status
+  
+  def initialize
+    @status = Array.new
+    @total = 0
+  end
+  
+  def add(item)
+    status << item
+    calculate_partial
+  end
+  
+  def check_order
+    calculate_basket
+    calculate_total
+  end
+  
+  def clear_order
+    self.status = Array.new
```

Comment:
Style/EmptyLiteral: Use array literal `[]` instead of `Array.new`. (https://github.com/bbatsov/ruby-style-guide#literal-array-hash)